packages/main/src/protocol-launcher.ts

```
@@ -18,19 +18,25 @@
    * @param url
    */
   sanitizeProtocolForExtension(url: string): string {
-    if (url.startsWith('podman-desktop://extension/')) {
-      url = url.replace('podman-desktop://extension/', 'podman-desktop:extension/');
-    } else if (url.startsWith('podman-desktop://preferences/experimental')) {
-      url = url.replace('podman-desktop://preferences/experimental', 'podman-desktop:experimental');
+    const doubleSlashPrefix = `${this.protocol}://`;
+    const singleColonPrefix = `${this.protocol}:`;
+
+    if (url.startsWith(`${doubleSlashPrefix}extension/`)) {
+      url = url.replace(`${doubleSlashPrefix}extension/`, `${singleColonPrefix}extension/`);
```

Biome (2.4.10)

[error] Assigning a function parameter is confusing.

(lint/style/noParameterAssign)